<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setu</class>
 <widget class="QWidget" name="Setu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setu</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>45</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>R18</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_4">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>20</y>
     <width>45</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>无AI</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>20</y>
     <width>45</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>横图</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_3">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>20</y>
     <width>45</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>竖图</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>20</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>20</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>20</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>原图</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelImage">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>700</width>
     <height>370</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelHint">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>700</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelTextBelow">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>470</y>
     <width>700</width>
     <height>110</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
